src/GafferScene/MeshSplit.cpp

```diff
@@ -54,33 +58,192 @@ using namespace IECoreScene;
 using namespace Gaffer;
 using namespace GafferScene;
 
-class MeshSplit::MeshSplitterData : public IECore::Data
+namespace {
+
+// Various private helpers for formatAsInternedString
+
+const std::string g_commaStr( ", " );
+
+template< typename T >
+void stringConvertHelper( const T &a, char *&current, char *end )
 {
+	// This interface was planned for std::to_chars, but it works fine with fmt::format_to_n
+	// too. It's all private anyway.
+	auto result = fmt::format_to_n( current, end - current, FMT_COMPILE("{}"), a);
+	if( result.size > (size_t)( end - current ) )
+	{
+		throw IECore::Exception( "Internal error in formatAsInternedString - buffer not large enough" );
+	}
+	current = fmt::format_to_n( current, end - current, FMT_COMPILE("{}"), a).out;
```

Comment:
Can we return `result.out` here, instead of calling `format_to_n()` again?

Reply:
Oh, that's an embarrassing mistake.  Yeah, it was absolutely my intention to use result.out, this was just a last minute fix to add the error handling, and made a copy-paste mistake.  It's interesting that this isn't even that obvious in the perf test - about a 3% slowdown, as opposed to 20% for snprintf or 40% for stringstream - does kinda validate that fmtlib is a good approach.